### PR TITLE
MIT License

### PR DESCRIPTION
**Pull request by alan-y**
License update and minor changes to `DESCRIPTION` to sort packages in alphabetical order.

### CODE REVIEW

File: DESCRIPTION
Comment (Moohan) [inline DESCRIPTION:1]: You can use `desc::desc_normalize()` - https://desc.r-lib.org/reference/desc_normalize.html - to do sort and organise the DESCRIPTION automatically.

File: DESCRIPTION [hunk 1]
```diff
@@ -3,30 +3,29 @@ Title: PHS R Templates
 Authors@R: c(
     person("Alan", "Yeung", email = "alan.yeung@phs.scot", role = c("aut", "cre"), comment = c(ORCID = "0000-0001-5226-3695")),
     person("Anna", "Price", email = "Anna.Price@Statistics.gov.uk", role = "aut"),
-    person("Rosalyn", "Pearson", email="rosalyn.pearson@phs.scot", role="aut")
+    person("Rosalyn", "Pearson", email = "rosalyn.pearson@phs.scot", role = "aut")
```
Comment (Moohan) [inline DESCRIPTION:6]: Adding me as a contributor 😉 
```suggestion
    person("Rosalyn", "Pearson", email = "rosalyn.pearson@phs.scot", role = "aut"),
    person("James", "McMahon", , "James.McMahon@phs.scot", role = "ctb",
           comment = c(ORCID = "0000-0002-5380-2029"))
```

File: DESCRIPTION [hunk 2]
```diff
@@ -3,30 +3,29 @@ Title: PHS R Templates
 Authors@R: c(
     person("Alan", "Yeung", email = "alan.yeung@phs.scot", role = c("aut", "cre"), comment = c(ORCID = "0000-0001-5226-3695")),
```
Comment (Moohan) [inline DESCRIPTION:4]: Adding PHS as the copyright holder - I've but the Data Science email but feel free to use whatever you think (or possibly none).
```suggestion
    person("Public Health Scotland", , , "phs.datascience@phs.scot", role = "cph"),
    person("Alan", "Yeung", email = "alan.yeung@phs.scot", role = c("aut", "cre"), comment = c(ORCID = "0000-0001-5226-3695")),
```

**Review by Moohan — APPROVED** [review]
Looks fine!

Maybe a task for the future to review if we still want to use `Depends: R (>= 3.2.0)` since that's a very old version, we don't use it in PHS and we don't test for it in the GH workflow either.

**Comment by alan-y** [comment]
> Looks fine!
> 
> Maybe a task for the future to review if we still want to use `Depends: R (>= 3.2.0)` since that's a very old version, we don't use it in PHS and we don't test for it in the GH workflow either.

Yeah think that was just copied by me a long time ago - not sure what people tend to put in there now or how to figure out the R version dependency properly...

**Pull request merged**